### ISSUE TITLE
Upload and replace renewed SSL Certificate offline

### ISSUE THREAD

**Issue by gotspatel** [FortiPower/PowerFGT#209]
Is there any option to fetch and replace a renewed SSL certificate from a local pc

OR

any option by which we can upload and replace a renewed SSL certificate from a local pc

**Comment by alagoutte** [comment]
Yes I think, it is possible to upload the certificate, only found the api call

**Comment by alagoutte** [comment]
Hi,

After a quick search, there is a doc page for this => https://docs.fortinet.com/document/fortigate/7.0.2/administration-guide/379103/uploading-a-certificate-using-an-api

you can use Invoke-FGTRestMethod for the payload, it should be work...

**Comment by alagoutte** [comment]
it is good ?
